### PR TITLE
Update Changelog in preparation for the 0.18.8 release

### PR DESCRIPTION
**Pull request by jcazevedo**
I have already set up a date because I plan on releasing this today.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +16,48 @@
 ### Fixed
 
 ### Security
+-->
+
+## [0.18.8] - 2023-04-27
+
+This release includes several dependency updates, some updates to make existing APIs more friendlier and a change to a
```
Comment (xRuiAlves) [inline CHANGELOG.md:23]: Grammar nit:

```suggestion
This release includes several dependency updates, some updates to make existing APIs friendlier and a change to a
```

or even

```suggestion
This release includes several dependency updates, some updates to make existing APIs more friendly and a change to a
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +16,48 @@
...
+```
+
+We have also promoted the SFTP file descriptor credentials from a tuple to a dedicated data type:
+
+```scala
+case class Credentials(host: String, port: String, auth: Either[Identity, String])
+```
+
+Additionally, the `Decoder` for [Squants](https://github.com/typelevel/squants)'s `Currency` was updated to become case
+insensitive, meaning that, for example, both `"usd"` and `"USD"` now decode to the same `USD` currency.
```
Comment (xRuiAlves) [inline CHANGELOG.md:46]: Perhaps a bit hard on the eye, too many commas. I would suggest something like:

```suggestion
insensitive. This means that, for example, both `"usd"` and `"USD"` now decode to the same `USD` currency.
```

**Review by xRuiAlves — APPROVED** [review]
Looks good, thanks!

Just left a couple minor comments

**Comment by xRuiAlves** [comment]
These `SimpleLruCacheSpec` tests seem to be failing transiently quite often, unfortunately :/
Had the plan (on the back of my mind) to eventually try and do something about it, but haven't found the time to be honest 😅

File: CHANGELOG.md (hunk 1 above)
Comment (jcazevedo) [reply CHANGELOG.md:23]: Yes, of course. At some point I had `more friendly` and wanted to go with just `friendlier`.

File: CHANGELOG.md (hunk 2 above)
Comment (jcazevedo) [reply CHANGELOG.md]: Yes. I agree.

**Review by xRuiAlves — APPROVED** [review]
Cool beans!

**Comment by jcazevedo** [comment]
> These `SimpleLruCacheSpec` tests seem to be failing transiently quite often, unfortunately :/ Had the plan (on the back of my mind) to eventually try and do something about it, but haven't found the time to be honest 😅

Yes. They are becoming quite annoying. I also have an item to address them in my todo list.

**Pull request merged**